lib/GLPI/Agent/Task/Inventory/Generic/PCI/Videos.pm

```diff
@@ -13,10 +13,7 @@ use GLPI::Agent::Tools::Generic;
 use constant    category    => "video";
 
 sub isEnabled {
-    # both windows and linux have dedicated modules
-    return
-        OSNAME ne 'MSWin32' &&
-        OSNAME ne 'linux';
+    return 1;
```

Comment:
Here you shouldn't enable this test if OSNAME is `MSWin32`: getPCIDevices() call requires `lspci` command on the system and it is not available on `MSWin32`.

About linux, there's a caveat. It can generate double entries as this is the case on my own system. If I run `glpi-inventory --partial video --debug --debug` I can control both `GLPI::Agent::Task::Inventory::Generic::PCI::Videos` & `GLPI::Agent::Task::Inventory::Linux::Videos` are run and the inventory reports:
```
      "videos": [
         {
            "chipset": "VGA compatible controller",
            "name": "NVIDIA Corporation TU106 [GeForce RTX 2060 SUPER]"
         },
         {
            "memory": 8192,
            "name": "NVIDIA GeForce RTX 2060 SUPER (TU106-A)",
            "resolution": "1920x1080"
         }
      ]
   },
```
If can confirm you I only have one `NVIDIA GeForce RTX 2060 SUPER` in my computer ;-)